common/src/main/java/com/wynntils/features/user/TranslationFeature.java

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.chat.RecipientType;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.core.features.properties.StartDisabled;
+import com.wynntils.core.services.TranslationManager;
+import com.wynntils.mc.utils.McUtils;
+import com.wynntils.wc.event.ChatMessageReceivedEvent;
+import com.wynntils.wc.event.NpcDialogEvent;
+import com.wynntils.wc.utils.WynnUtils;
+import net.minecraft.network.chat.TextComponent;
+import net.minecraftforge.eventbus.api.EventPriority;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+@StartDisabled
+public class TranslationFeature extends UserFeature {
+    public static TranslationFeature INSTANCE;
+
+    @Config
+    public String languageName = "";
```

Comment:
Is this fine as empty? Perhaps make it null?

Reply:
It is fine as empty. In general, my principle is to prefer the empty string before null for strings, and to prefer the empty set/list/map before null for set/list/map. 

But maybe I should put in an explicit check so I don't rely on the translation API to return an error. (Or maybe I already do, don't remember; that code is stolen directly from Legacy)